src/OpenDiffix.Core/Aggregator.fs

```diff
@@ -171,7 +171,7 @@ type private DiffixCount(minCount) =
         | Anonymizer.Result.Count value -> Integer(max value minCount)
 
 type private DiffixCountDistinct(minCount) =
-  let mutable aidsCount = 0
+  let mutable aidsCount = Option<int>.None
```

Comment:
My problem with `0` is that it conflates:
1. an invalid situation of 0 AID instances
2. non-initialized instance of `DiffixCountDistinct`

this particular change clarifies this, and makes the check in `Final` cleaner.